A little bug: user name missing in the composing message

mmoqui opened the pull request:
Fix the following bug: when one user is writing a message, its name is missing in the composing message in the chat/groupchat window of the other(s) user(s)

sualko commented:
We should find a way to add the name only in rooms, because in a 2 person conversation this is not needed.

mmoqui commented:
I updated the code so that the name of the user(s) composing a message is(are) displayed only for groupchat. I rebased my changes to have only one commit.

sualko commented:
> I rebased my changes to have only one commit.

No need, because this can easily be done via githubs web ui. Thanks for your effort.
